<?xml version="1.0" encoding="utf-8"?>
<report
	progress="">
	<activity
		result="Failed"
		duration="12.18s"
		durationms="12184"
		type="root"
		rid="2a08e58f2e34bf3"
		totalerrorcount="1"
		totalwarningcount="0"
		totalsuccesscount="0"
		totalfailedcount="0"
		totalblockedcount="0"
		user="Karun Shrestha"
		host="THINK-PK2N901"
		rxversion="6.1.0.0"
		osversion="Windows 7 Service Pack 1 64bit"
		runtimeversion="4.0.30319.42000"
		procarch="32bit"
		language="en-US"
		screenresolution="3520x1080"
		timestamp="1/10/2017 1:46:04 PM"
		timestampiso="2017-01-10T18:46:04"
		timeoutfactor="1"
		module="ValidateTableTest">
		<detail>
			ValidateTableTest
		</detail>
		<varbindings/>
		<item
			timeRelativeToTestSuiteStartTime="00:12.183"
			timeRelativeToTestModuleStartTime="00:12.183"
			timeWallClock="13:46:16"
			time="00:12.183"
			level="Error"
			category="Module">
			<message>
				The test run has been aborted by the user.
			</message>
			<metainfo
				loglvl="Error"/>
		</item>
	</activity>
</report>
